bot/src/handler.rs

```diff
@@ -72,7 +73,20 @@ where
     }
 
     #[instrument(skip_all)]
+    #[allow(unused_must_use)]
     async fn message(&self, ctx: Context, new_message: Message) {
+        // prevent sending of default stickers
+        if !new_message.is_private() {
+            for sticker in &new_message.sticker_items {
+                if sticker.format_type == StickerFormatType::Lottie {
+                    new_message.author.direct_message(&ctx, |m| m.content("fuck off")).await;
```

Comment:
```suggestion                  
```

Reply:
This is really just a placeholder for and I was thinking of creating a reusable message to notify the user in case of sanctions.
Not sure how to generate human readable text though

Reply:
(This is probably not worth the effort but whatever)

Reply:
nvm, I did it